python_modules/libraries/dagster-k8s/dagster_k8s/ops/k8s_job_op.py

```diff
@@ -416,6 +416,16 @@ def execute_k8s_job(
             num_pods_to_wait_for=num_pods_to_wait_for,
         )
     except (DagsterExecutionInterruptedError, Exception) as e:
+        try:
+            pods = api_client.get_pod_names_in_job(job_name=job_name, namespace=namespace)
+            pod_debug_info = "\n\n\n\n".join([api_client.get_pod_debug_info(pod_name, namespace) for pod_name in pods])
+        except Exception:
+            context.log.info(
+                f"Error trying to get pod debug information for failed k8s job {job_name}"
+            )
+        else:
+            context.log.error(f"Pod debug info:\n\n{pod_debug_info}")
```

Comment:
can this log message give a bit more context about why its being logged?

"Debug information for failed k8s job {job name}:\n\n{pod_debug_info}"

Reply:
updated